src/utils.js

```diff
@@ -95,7 +96,8 @@ const utils = {
     if (!origin) {
       throw Error(`No proxy strain: ${ctx.strain.name}`);
     }
-    const url = `${origin.useSSL ? 'https' : 'http'}://${origin.hostname}:${origin.port}${req.path}`;
+    const proxyPath = path.posix.join('/', origin.path, ctx.relPath);
```

Comment:
Maybe it is this change that caused https://github.com/adobe/helix-cli/issues/878 ?